tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,77 @@
+"""Task 1 and 2 of 2nd Module."""
+
+
+# Globals
+# TODO: move globals to separate setting module?
+EXCLUDED_ITEMS = ['chewed gum', 'rubbish', 'used tissue']
+INVENTORY_LIGHT_WEIGHT = 60
+INVENTORY_HEAVY_WEIGHT = 70
+INVENTORY_MAX_WEIGHT = 80
+
+
+def display_inventory(inventory: dict):
+    """Display hero's inventory.
+
+    Arguments:
+        inventory   -- {item_name: item_num}
+    """
+    print('Inventory:')
+    item_total = 0
+    for item_name, item_count in inventory.items():
+        print(item_count, item_name)
+        item_total += item_count
+
+    # Print weight capacity
+    print(f'\nTotal number of items: {str(item_total)}\n')
+    if INVENTORY_LIGHT_WEIGHT <= item_total < INVENTORY_HEAVY_WEIGHT:
+        print('CAUTION: Your backpack weighs a lot, '
+              'your stamina runs out quicker!')
+    elif INVENTORY_HEAVY_WEIGHT <= item_total < INVENTORY_MAX_WEIGHT:
+        print('CAUTION: Your equipment is very heavy, '
+              'you\'re moving slower than usual!')
+    elif item_total >= INVENTORY_MAX_WEIGHT:
+        print('CAUTION: You are overloaded, can\'t move!')
+
+
+def add_to_inventory(inventory: dict, added_items) -> dict:
```

Comment:
It's good that you already use type hints. You may want to add it also for `added_items`.